TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift

```diff
@@ -33,36 +50,33 @@ class PushFactory {
   private let repository: FactorProvider
   private let keyStorage: KeyStorage
   
-  init(repository: FactorProvider, keyStorage: KeyStorage = KeyStorageAdapter()) {
+  init(repository: FactorProvider, keyStorage: KeyStorage) {
     self.repository = repository
     self.keyStorage = keyStorage
   }
 }
 
 extension PushFactory: PushFactoryProtocol {
-  func createFactor(withAccessToken accessToken: String, friendlyName: String, pushToken: String, serviceSid: String,
-                    identity: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
+  func createFactor(withAccessToken accessToken: String, friendlyName: String, serviceSid: String,
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/9cbd540c6135af3d1641ad17aef5ccc5e04d146c/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L60"></span>Function should have 7 parameters or less: it currently has 8<br />
<code>function_parameter_count</code> <code>PushFactory.swift:60</code></td>
    </tr>
  </tbody>
</table>